archivers/libzzip/Portfile

```diff
@@ -18,28 +18,29 @@ long_description    The ZZIPlib provides read access on ZIP-archives. The \
                     access files being either real files or zipped files, \
                     both with the same filepath.
 
-checksums           rmd160  d401fc327138d5b997c0cd318f808f95cea1f937 \
-                    sha256  c5cf1c11a6019862e5b6daea4d7c72124affe687b9637effc7ad3caf2b7b1d96 \
-                    size    1132209
+checksums           rmd160  20b12141e49c6332d8fd8b610d1fa4a868d9072b \
+                    sha256  61a14917400dbef5eafb482ddc862f3edad86c888a73f445f82e1ab856d24b46 \
+                    size    1162325
 
-depends_build       port:pkgconfig \
-                    port:python27
+set python.branch   3.9
+set python.version  [join [split ${python.branch} "."] ""]
+set python.prefix   ${frameworks_dir}/Python.framework/Versions/${python.branch}
+set python.bin      ${python.prefix}/bin/python${python.branch}
 
-depends_lib         port:zlib
+depends_build-append \
+                    port:pkgconfig \
+                    port:python${python.version}
 
-set docdir ${prefix}/share/doc/${name}
+depends_lib-append  port:zlib
 
-configure.python    ${prefix}/bin/python2.7
-configure.env       ac_cv_path_PAX=:
-
-post-configure {
-    set builddir [glob -dir ${worksrcpath} "*apple-darwin*"]
-    reinplace -E {s|-arch [a-z0-9_]+||g} \
-        ${builddir}/zzip/zziplib-uninstalled.pc \
-        ${builddir}/zzip/zziplib.pc
-}
+configure.args-append \
+                    -DPYTHON_EXECUTABLE=${python.bin} \
```

Comment:
is just doing `${prefix}/bin/python${python.branch}` not sufficient? That would avoid the need to define `python.prefix` and `python.bin`